readthedocs/organizations/signals.py

```diff
@@ -56,28 +52,13 @@ def remove_organization_completely(sender, instance, using, **kwargs):
     # to be sure that the projects we are deleting here belongs only to the
     # organization deleted
     projects = Project.objects.annotate(
-        Count('organizations'),
-    ).filter(
-        organizations__in=[organization],
-        organizations__count=1,
-    )
-
-    versions = Version.objects.filter(project__in=projects)
-    teams = Team.objects.filter(organization=organization)
-    team_invites = TeamInvite.objects.filter(organization=organization)
-    team_memberships = TeamMember.objects.filter(team__in=teams)
-
-    # Bulk delete
-    team_memberships.delete()
-    team_invites.delete()
-    teams.delete()
-
-    # Granular delete that trigger other complex tasks
-    for version in versions:
-        # Triggers a task to remove artifacts from storage
-        version.delete()
```

Comment:
Deleting the project will free all resources linked to its versions

Reply:
Does this delete the users subscription from Stripe? I think we've hit issues with this in the past. I think those were users who deleted, but didn't delete the organization, which this will fix 👍

Reply:
In case it "deletes" the subscription from Stripe, is it possible to add an automated comment on the subscription? Like "Canceled because the user deleted their Read the Docs account" or similar.

Reply:
if you mean deleting the subscription from the org, yes, we already do that when the org is deleted

https://github.com/readthedocs/readthedocs.org/blob/04d20f34c058e2916cd1b796b0a85b4c4b75836e/readthedocs/subscriptions/signals.py#L57-L58